src/api/resources/collect_record.py

```diff
@@ -134,9 +122,7 @@ def missing(self, request, *args, **kwargs):
         sql, params = qs.query.get_compiler(using=qs.db).as_sql()
 
         query_sql = (
-            "SELECT * FROM unnest(%s) _pk_ "
-            "EXCEPT ALL "
-            "SELECT _pk_ FROM (" + sql + ") as foo"
+            "SELECT * FROM unnest(%s) _pk_ " "EXCEPT ALL " "SELECT _pk_ FROM (" + sql + ") as foo"
```

Comment:
The double quotes are preserved in eliminating multiple lines -- maybe we should look through all changed sql for this?
And/or: maybe this particular case should be ignored? Doesn't seem more readable to me.